demo/src/app/components/carousel/carousel.component.ts

```diff
@@ -8,6 +8,7 @@ import {DEMO_SNIPPETS} from './demos';
       <ngbd-api-docs directive="NgbCarousel"></ngbd-api-docs>
       <ngbd-api-docs directive="NgbSlide"></ngbd-api-docs>
       <ngbd-api-docs-config type="NgbCarouselConfig"></ngbd-api-docs-config>
+      <ngbd-api-docs-class type="NgbSlideCarouselEvent"></ngbd-api-docs-class>
```

Comment:
I don't think this new functionality warrants a separate demo - 2 carousel side by side already look wired on our demo page so I would prefer adding more demos unless those really add a lot of value.